projects/buttons/fastener/scripts/start-button.sh

```diff
@@ -116,6 +120,9 @@ else
 /fusion/bin/webapps stop && rm /fusion/var/webapps/webapps/* && /fusion/bin/webapps start
 fi
 
+# Put the security.json file on Zookeeper
+/fusion/$FUSION_VERSION/apps/solr-dist/server/scripts/cloud-scripts/zkcli.sh -zkhost localhost:9983/lwfusion/${FUSION_VERSION/solr -cmd putfile /security.json security.json
```

Comment:
${FUSION_VERION - doesn't have closing curly bracket!

Reply:
oof. Trying to do this VIM thing without plugins for syntax ends today.